src/main/java/com/hms_networks/americas/sc/extensions/time/LocalTimeOffsetCalculator.java

```diff
@@ -56,65 +30,72 @@ public static synchronized long getLocalTimeOffsetMilliseconds() {
   /**
    * Calculates the offset between UTC time and local time (in milliseconds) and stores the result
    * to be returned by {@link #getLocalTimeOffsetMilliseconds()}. The result is also returned by
-   * this method. The local time offset is calculated using an on-device HTML file that is accessed
-   * using the default Ewon username and password, or an updated username/password combination
-   * specified by calling {@link #setDeviceUsername(String)} and {@link #setDevicePassword(String)}.
-   * The username/password combination must be updated to reflect a valid username/password
-   * combination on the Ewon Flexy device.
+   * this method. The local time offset is calculated using a script expression export block
+   * descriptor call.
```

Comment:
This comment is a little confusing.
```suggestion
   * Reads the offset between UTC time and local time (in milliseconds) via export block descriptor. 
   * Stores the result which can be read by {@link #getLocalTimeOffsetMilliseconds()}.
   * Note: Implementation changed in 1.4 from HTTP request to export block descriptor.  
```

Reply:
Modified this comment a bit